lib/Listener/LoginMailListener.php

```diff
@@ -40,21 +42,25 @@ class LoginMailListener implements IEventListener {
 
 	/** @var ILogger */
 	private $logger;
+	
 	/** @var IMailer */
 	private $mailer;
+	
 	/** @var IUserManager */
 	private $userManager;
+	
 	/** @var IL10N */
 	private $l;
+	
+	/** @var IConfig */
+	protected $config;
```

Comment:
```suggestion
	private IConfig $config;
```

Reply:
Let's not get caught up in minor details.

Reply:
Thanks for the note. I'll try to keep in mind next time. :)